libevmasm/Assembly.h

```diff
@@ -169,6 +171,10 @@ class Assembly
 
 	unsigned bytesRequired(unsigned subTagSize) const;
 
+	static AssemblyItem assemblyItemFromJSON(Json::Value const& _json, Assembly& _assembly);
+	static void forEachJSONAssemblyItem(Json::Value const& _code, Assembly& _assembly);
```

Comment:
`forEachJSONAssemblyItem()` looks like an implementation detail that should be private.